### DISCUSSION TITLE
:open-output

### DISCUSSION THREAD

**Discussion by tobyfrancis** [helix-editor/helix#9231] [Ideas]
Allowing shell outputs to pass filepaths to Helix's :output would open the door (pun intended) for shell scripts to trigger Helix to open a file (my primary use-case is opening the first file that fails to compile or fails a test in large projects, as this can be tedious to find). I'm not sure if this is already possible, I may just be ignorant here.  The only way I see this possible currently is quite a bit of tmux/zellij magic that could be drastically simplified with such a built-in command.

Intended usage:
`:open-output my_script_of_choice_which_writes_filepath_to_stdout`

Thank you so much for this wonderful project that has helped tremendously in my software engineering career!

**Comment by kirawi** [comment]
This is addressed by https://github.com/helix-editor/helix/pull/6979 via `:open %sh{<COMMAND>}`
